database/data.sql

```diff
@@ -1,3 +1,6 @@
 insert into "accounts" ("username", "photoUrl", "currentRating", "happyLevel")
-values ('John Smith', '/images/example1.jpg', 3, 'angry'),
-('Jane Smith', '/images/example2.jpg', 3.5, 'content');
+values ('John Smith', '/images/Blue.png', 3, 'angry'),
+('Jane Smith', '/images/example.jpg', 3.5, 'content'),
+('Hippo Smith', '/images/example1.png', 1, 'angry'),
+('Bro Smith', '/images/example2.jpg', .5, 'angry'),
+('Foo Smith', '/images/example3.png', 5, 'happy');
```

Comment:
If the face icon we're using is based on their rating, then we can leave it out of the database and just write logic in the UI to pick the face based on the rating.